<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>云主机设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_listname">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>主机列表：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbx_hostlist">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>UHost</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_updatelist">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/update_64px.png);
background-attachment: fixed;
background-origin:content-box;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_shutdown">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>181</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>关机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reboot">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>181</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>重启</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_poweroff">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>221</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>强制掉电</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_poweron">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>221</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reinstall">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>181</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>重装系统</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pubkey">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pvtkey">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>公钥：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>私钥：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>221</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
